### DISCUSSION TITLE
Dark Mode?

### DISCUSSION THREAD

**Discussion by Jacob-077** [AntiMicroX/antimicrox#394] [Q&A] [answered]
Is there's a way to enable the dark mode on windows?
There's a disclaimer on the main GitHub page that states: "Theme may depend on your system configuration."
Does that mean dark mode isn't available on windows?

**Comment by pktiuk** [comment, marked as answer]
AFAIK Windows doesn't support system-wide theming for applications (like Linux or Android)   
But there are some plans to tweak the default Windows theme to make it a bit darker.

**Reply by pktiuk** [reply]
Linked issue: https://github.com/AntiMicroX/antimicrox/issues/490
